Backpack/Card/Classes/BPKDividedCard.m

```diff
@@ -0,0 +1,121 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#import "BPKDividedCard.h"
+#import "BPKCardDivider.h"
+
+#import <Backpack/Spacing.h>
+#import <Backpack/Color.h>
+#import <Backpack/Radii.h>
+#import <Backpack/Shadow.h>
+
+NS_ASSUME_NONNULL_BEGIN
+@interface BPKDividedCard()
+@property ( strong, nonatomic ) UIStackView *contentView;
```

Comment:
Please write properties as `@property(strong, nonatomic) Type * name`. I.e. remove the extra spaces before the opening parenthesis and inside the pair